src/components/views/messages/MPollBody.tsx

```diff
@@ -484,7 +484,7 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
         const totalVotes = this.totalVotes(votes);
         const winCount = Math.max(...votes.values());
         const userId = this.context.getUserId();
-        const myVote = userVotes?.get(userId)?.answers[0];
+        const myVote = userVotes?.get(userId!)?.answers[0];
```

Comment:
We can use `!` here because a `Map` lookup with `null` or `undefined` will still give an `undefined` `myVote` which is acceptable in the logic below.